tests/cypress/integration/reportRepair/summary.spec.js

```diff
@@ -6,206 +6,232 @@ import {
   intercept_repair_triage,
   navigateToPageSelectRadioOptionAndContinue,
   navigateToSummaryPage,
+  navigateToCommunalSummaryPage,
 } from '../../support/helpers';
 
 describe('summary', () => {
-  let timeSlotValueFunction;
-  const address = '1 Downing Street, London, SW1A 2AA';
-  const repairDescription = 'Eius postea venit saepius arcessitus.'
-  const phoneNumber = '02085548333';
+  describe('tenant summary', () => {
+    let timeSlotValueFunction;
```

Comment:
Would specifying the `REPAIR_TYPE` environment variable here remove the need for a global definition?
```suggestion
    env: {
      REPAIR_TYPE: "tenant",
    },
    let timeSlotValueFunction;
```

Reply:
We specifically set it in this test but we needed the facility to run the logic from tests that don't specifically override the env variable, meaning that we need this globally with a default value of tenant.

Reply:
Update - I've tested removing it globally and it works. If not defined for the tests you are running then it just comes back as blank.